### PR TITLE
🎃 Clean-up examples

### PR DESCRIPTION
**Pull request by marekkirejczyk**

### CODE REVIEW

File: packages/example/src/pages/Block.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import { useBlockMeta, useBlockNumber, useEthers } from '@usedapp/core'
+
+export function Block() {
+  const blockNumber = useBlockNumber()
+  const { chainId } = useEthers()
+  const { timestamp, difficulty } = useBlockMeta()
+  return (
+    <div>
+      <p>Chain id: {chainId}</p>
+      <p>Current block: {blockNumber}</p>
```
Comment (rzadp) [inline packages/example/src/pages/Block.tsx:11]: Why not `{blockNumber && (...)` like the rest?

**Review by rzadp — APPROVED** [review]
Looks good to me.

**Review by Marik-D — APPROVED** [review]

File: packages/example/src/pages/Tokens.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+}
+
+export function Tokens() {
+  const { chainId, activateBrowserWallet, deactivate, account } = useEthers()
+  const tokenList = getTokenList(chainId)
+  const balances = useTokensBalance(tokenList, account)
+
+  return (
+    <div>
+      {account && <button onClick={() => deactivate()}>Disconnect</button>}
```
Comment (Marik-D) [inline packages/example/src/pages/Tokens.tsx:31]: Inline lambda

File: packages/example/src/pages/Balance.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+export function Balance() {
+  const { activateBrowserWallet, deactivate, account } = useEthers()
+  const userBalance = useEtherBalance(account)
+  const stakingBalance = useEtherBalance(STAKING_CONTRACT)
+
+  return (
+    <div>
+      {!account && <button onClick={() => activateBrowserWallet()}>Connect</button>}
+      {account && <button onClick={() => deactivate()}>Disconnect</button>}
```
Comment (Marik-D) [inline packages/example/src/pages/Balance.tsx:15]: inline lambda

**Pull request merged**